### PR TITLE
Slides for class 16

### PR DESCRIPTION
**Pull request by alena456**

### CODE REVIEW

File: src/lessons/16_api_calls/api_calls.slides.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>API Calls</title>
+  <meta name="date" content="2021-05-25" />
```
Comment (oktupol) [inline src/lessons/16_api_calls/api_calls.slides.html]: Once again 😉 
```suggestion
  <meta name="date" content="2021-05-27" />
```

File: src/lessons/16_api_calls/api_calls.slides.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                                .then(response => console.log(response.json()));
+                        </code>
+                        </pre>
+          <p class="fragment">
+            <a href="https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API" class="small">More on fetch</a>
+          </p>
+        </section>
+        <section>
+          <h2>{JSON} Placeholder</h2>
+          <a href="https://jsonplaceholder.typicode.com">Check it out!</a>
```
Comment (oktupol) [inline src/lessons/16_api_calls/api_calls.slides.html:86]: Oh wow, this is neat!

**Review by oktupol — COMMENTED** [review]
Looks good to me, except for the date

File: src/lessons/16_api_calls/api_calls.slides.html (hunk 1 above)
Comment (alena456) [reply src/lessons/16_api_calls/api_calls.slides.html]: Damn I thought I fixed it this time ^^

**Review by oktupol — APPROVED** [review]

**Pull request merged**